packages/graphback-cli/src/commands/init.ts

```diff
@@ -0,0 +1,57 @@
+import * as execa from 'execa'
+import { lstatSync, readdirSync } from 'fs'
+import { prompt as ask } from 'inquirer'
+import { ncp } from 'ncp'
+import { join } from 'path'
+import { promisify } from 'util'
+import * as yargs from 'yargs'
+import { logInfo } from '../utils'
+
+const copyAsync = promisify(ncp)
+
+type Params = { name?: string }
+
+function getTemplateDir(): string {
+  return `${__dirname.split('packages')[0]}templates`
```

Comment:
This is not going to work! This command line will not have access to the templates. 
We need to download them from github!